libkpod/remove.go

```diff
@@ -0,0 +1,46 @@
+package libkpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/oci"
+)
+
+// Remove removes a r
+func (c *ContainerServer) Remove(container string, force bool) (string, error) {
```

Comment:
Also, can we name this ContainerRemove? We're eventually going to have a PodRemove, too